packages/terra-table/src/Table.jsx

```diff
@@ -308,7 +310,25 @@ function Table(props) {
     return (hasSelectableRows ? [tableRowSelectionColumn] : []).concat(pinnedColumns).concat(overflowColumns);
   }, [hasSelectableRows, intl, onRowSelectionHeaderSelect, overflowColumns, pinnedColumns]);
 
-  const [tableColumns, setTableColumns] = useState(displayedColumns.map((column) => initializeColumn(column)));
+  // Create new displayedColumns object to pass to Section sub component to account for column spans
+  const displayedColumnsWithColumnSpan = [];
```

Comment:
Do we need both displayedColumns and displayedColumnsWithColumnSpans?  It seems like we could have one object.

Reply:
Removed this from Table code and using the one created at DataGrid Level here.